app/assets/stylesheets/hyrax/_collections.scss

```diff
@@ -261,4 +244,280 @@ button.branding-banner-remove:hover {
 
 .delete-admin-set-collection-form {
   text-align: left;
-}
\ No newline at end of file
+}
+
+// Issue 1428 Styles (DT)
```

Comment:
A better comment is 'Styles for collections public landing page'.  Issues typically are not referenced in code unless it is part of a TODO comment.